gpt_engineer/main.py

```diff
@@ -51,13 +51,12 @@ def chat(
         logs=DB(memory_path / "logs"),
         input=DB(input_path),
         workspace=DB(workspace_path),
-        identity=DB(app_dir / "identity"),
+        identity=DB(Path(os.path.curdir) / "identity"),
     )
 
     for step in STEPS[steps_config]:
         messages = step(ai, dbs)
         dbs.logs[step.__name__] = json.dumps(messages)
 
-
 if __name__ == "__main__":
-    app()
+    app.run()
```

Comment:
I get an error on this -  says typer run() does not exist

Reply:
Yeah I guess that's why it's giving an error. I need to find another solution for OPENAPI error

Reply:
Yeah, I don't think this works for me either.

Also, I think your branch is not up to date with `main` @subnwa

Reply:
I realized that there is an error in my own system, I will review it again.